src/Components/WebAssembly/Server/src/DebugProxyLauncher.cs

```diff
@@ -168,6 +168,7 @@ void OnOutputDataReceived(object sender, DataReceivedEventArgs eventArgs)
                 if (match.Success)
                 {
                     capturedUrl = match.Groups["url"].Value;
+                    capturedUrl = capturedUrl.Replace("http://", "ws://");
```

Comment:
Should we also also include the wss? 
```suggestion
                    capturedUrl = capturedUrl.Replace("http://", "ws://");
                    capturedUrl = capturedUrl.Replace("https://", "wss://");
```

Also any chance this has an unescaped http:// as part of the query / route?

Reply:
I don't think we need https:// because BrowserDebugProxy is always started as http, right?
https://github.com/dotnet/runtime/blob/59e643b9932122a12e71ceb360a9ae3bb9420dec/src/mono/wasm/debugger/BrowserDebugHost/Program.cs#L36

Reply:
I don't think so the query/route part will be like this: ws%3A%2F%2F127.0.0.1%3A53061%2Fdevtools%2Fbrowser%2F2979b015-4625-bea7-2071-e9b04f9ca35c